samwise.js

```diff
@@ -12,95 +12,90 @@
     var autoplayDefault = 0;
```

Comment:
SCRIPT-I appreciate how well-structured and clean your code is. It's very easy to follow!